src/JsonPatch.php

```diff
@@ -71,6 +71,13 @@ public static function import(array $data)
                 throw new MissingFieldException('path', $operation);
             }
 
+            if (!is_string($operation->op)) {
+                throw new Exception('Invalid field type - "op" should be a string');
```

Comment:
Could throw a new `InvalidFieldTypeException` instead? Probably depends on if we want to handle these errors differently.